samples/nrf9160/cloud_client/src/main.c

```diff
@@ -154,6 +186,10 @@ void main(void)
 		printk("dk_buttons_init, error: %d\n", err);
 	}
 #endif
+	printk("Waiting for LTE link...\n");
+	k_sem_take(&lte_connected, K_FOREVER);
+
+	printk("Got semaphore\n");
```

Comment:
maybe not needed. or should be more descriptive

Reply:
Changed to inform that the LTE link is up